tailwind/plugins/theme/gc-colors.js

```diff
@@ -13,13 +13,5 @@ module.exports = function () {
     slate: '#4E4B48',
     'gc-sky': '#4287BD', // For homepage hero sky gradient
     'gc-black': '#17171A',
-    // Colors inherited from the SAA project
-    'digital-red': {
-      xlight: '#F83535', // Passed contrast test for black background
-    },
-    'cardinal-red': {
-      xdark: '#7A0000', // Passed contrast test with digital-red-xlight as text
-      xxdark: '#541107', // Used for hover/focus color for xdark
-    },
```

Comment:
I added these colors to Decanter 7.1.1 since we use them so often for a11y contrast